src/app/components/forms/post/_community-pill/selector/scroll/scroll.vue

```diff
@@ -0,0 +1,5 @@
+<template>
```

Comment:
Instead of this, since there's no template or styles, just make the TS file a non-default class export with a `render()` function that returns `h('div')`. Example file: `src/app/components/client/system-progress/system-progress.ts`